src/android/toga_android/widgets/selection.py

```diff
@@ -1,10 +1,14 @@
+from rubicon.java import JavaClass
 from travertino.size import at_least
 
 from ..libs.android import R__layout
+from ..libs.android.util import TypedValue
 from ..libs.android.view import Gravity, View__MeasureSpec
-from ..libs.android.widget import ArrayAdapter, OnItemSelectedListener, Spinner
+from ..libs.android.widget import OnItemSelectedListener, Spinner
 from .base import Widget, align
 
+BeewareSpinnerAdapter = JavaClass("org/beeware/android/BeewareSpinnerAdapter")
```

Comment:
We don't need to replicate "Beeware" in this class name; `org.beeware.android.SpinnerAdapter` is sufficiently unique.

Reply:
I've done that because there already **is** a `SpinnerAdapter` in Android - and it is not a class, but an Interface.
Of course, technically, it would still work because we are in a different namespace, but I thought it would be less confusing if I use a new name.

Reply:
The template is not **really** Toga specific. 
The new Class could be used by any GUI. It just provides the necessary features to control the (size of the) font - which, in my opintion, is a short-coming of the standard implementation.